tests/parser/cellml_1_0.cpp

```diff
@@ -0,0 +1,145 @@
+/*
+Copyright libCellML Contributors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+#include "test_utils.h"
+
+#include "gtest/gtest.h"
+
+#include <libcellml>
+
+TEST(ParserTransform, emptyCellml10)
+{
+    const std::string e =
```

Comment:
Very minor, but shouldn't in be `in` rather than `e` that we tend to use for an expected output? (Same in the other tests.)